test/strategy.test.ts

```diff
@@ -39,6 +39,9 @@ function callGetScores(example) {
   );
 }
 
+const now = Date.now();
+Date.now = jest.fn(() => now);
+
```

Comment:
what is this for? 🤔

Reply:
Explained into the description of the ticket

Reply:
You added a test for the check "Voting power should not depend on other addresses", which is recomputing the distribution. However, we use Date.now() in the strategy, so the result is different. To fix it, I suggest to mock Date.now() to return always the same timestamp

Reply:
We could have had this choice also in the contract:

```solidity
 if(isSupplyDisabled) revert OperationDisabled();
```
With the same error for each operation
But the design of the contract is different (one error per operation), and IMO for clear understanding of morpho-js, it is a good thing to reproduce the errors of the Morpho contract.